directories/src/config/index.js

```diff
@@ -17,7 +17,7 @@ export const PEER_PORT = 8000;
 export const PEER_PATH = "/peer";
 export const PEER_DEBUG = 3;
 
-export const IOTA_API_PROVIDER = "https://nodes.thetangle.org:443";
+export const IOTA_API_PROVIDER = "http://18.188.64.13:14265";
```

Comment:
For future reference, it's better not to check these kind of changes in.  But it's only a test broker so it's not a big deal.

Reply:
I forgot about the test broker - Is there an automated way of replacing these for git commits?